src/app/server/CommissioningWindowManager.cpp

```diff
@@ -84,7 +84,7 @@ void CommissioningWindowManager::OnPlatformEvent(const DeviceLayer::ChipDeviceEv
     }
     else if (event->Type == DeviceLayer::DeviceEventType::kOperationalNetworkEnabled)
     {
-        app::DnssdServer::Instance().AdvertiseOperational();
+        LogErrorOnFailure(app::DnssdServer::Instance().AdvertiseOperational());
```

Comment:
And what about the progress log below?

Reply:
I am actually also re-considering this PR:  apparently for thread it is normal to get an error here because SRP is still being initialized. This may make spammy logs, unsure if good (it should help us not treat failures as normal) or annoying.

Will update logic though to not use LogErrorOnFailure due to usage of __FILE__